src/file-utils/edfi_lms_file_utils/file_repository.py

```diff
@@ -5,61 +5,85 @@
 
 import os
 from typing import Callable, List, Optional
+from dataclasses import dataclass
 
 
 import edfi_lms_file_utils.directory_repository as dr
 
 
-def _scan_files(directory: Optional[str]) -> Optional[List]:
-    if directory is None or not os.path.exists(directory):
-        return None
+@dataclass
+class FileInfo:
+    path: str
+    name: str
+    size: int
+
+    def has_contents(self) -> bool:
+        """ "
+            Determines if a file has contents beyond a couple of line breaks; this situation occurs when
```

Comment:
There's an extra quotation mark and something happened with the spacing.
```suggestion
        """ 
        Determines if a file has contents beyond a couple of line breaks; this situation occurs when
```